reference/5.1/Microsoft.PowerShell.Core/About/about_Arrays.md

```diff
@@ -617,12 +612,12 @@ Where(scriptblock expression[, WhereOperatorSelectionMode mode
 > the scriptblock is the only parameter. Also, there must not be a space
 > between the method and the opening parenthesis or brace.
 
-The `Expression` is scriptblock that is required for filtering, the `mode`
+The `Expression` is a scriptblock that's required for filtering, the `mode`
 optional argument allows additional selection capabilities, and the
 `numberToReturn` optional argument allows the ability to limit how many items
 are returned from the filter.
 
-The acceptable values for `mode` are:
+The value of `mode` must be a [WhereOperatorSelectionMode][02] enum value:
 
 - Default (0) - Return all items
```

Comment:
Nit: Should these values be backticked since you can pass them explicitly? i.e.

```suggestion
- `Default` (`0`) - Return all items
```

> **Note**: I can't select the full list for suggestion because the rest is too far from changes.